genai-perf/tests/test_telemetry_data_collector.py

```diff
@@ -143,3 +143,44 @@ def test_collect_metrics(
                 self.triton_metrics_response
             )
             mock_sleep.assert_called_once()
+
+    @patch("requests.get")
+    def test_url_reachability_check_success(
+        self,
+        mock_get: MagicMock,
+        collector: MockTelemetryDataCollector,
+    ) -> None:
+        mock_get.return_value.status_code = requests.codes.ok  # 200
+        assert collector.check_url_reachability() is True
+
+    @patch("requests.get")
+    def test_url_reachability_check_failure(
+        self, mock_get: MagicMock, collector: MockTelemetryDataCollector
+    ) -> None:
+        # Simulate a 404 Not Found error
```

Comment:
Are the number important here? I feel like we could remove all of the comments, as the request codes names are clear and have associated documentation.

Reply:
Missed it sorry